bench-tps/src/bench.rs

```diff
@@ -383,9 +418,22 @@ where
         use_durable_nonce,
         instruction_padding_config,
         num_conflict_groups,
+        load_accounts_from_address_lookup_table,
         ..
     } = config;
 
+    // if --load_accounts_from_address_lookup_table is used, creates Lookup Table account,
+    //     then extend it with spepcified number of account.
+    // All bench transfer transactions will include a `sbf_program_id` instruction to load
+    //     specified number of accounts from Lookup Table account as writable accounts.
+    let lookup_table_address =
+        load_accounts_from_address_lookup_table.map(|(sbf_program_id, number_of_accounts)| {
+            let lookup_table_account =
+                create_address_lookup_table_account(client.clone(), &id, number_of_accounts)
```

Comment:
seems fine to do this for local tests, but I'd think we don't want to create and leak the account everytime on testnet.

Can/should we do some form of stashing here? I think we do it with the regular accounts.